src/render/opengles2/SDL_render_gles2.c

```diff
@@ -663,26 +663,31 @@ static int
 GLES2_QueueDrawPoints(SDL_Renderer * renderer, SDL_RenderCommand *cmd, const SDL_FPoint * points, int count)
 {
     const SDL_bool colorswap = (renderer->target && (renderer->target->format == SDL_PIXELFORMAT_ARGB8888 || renderer->target->format == SDL_PIXELFORMAT_RGB888));
-    int color;
     const size_t vertlen = (2 * sizeof (float) + sizeof (int)) * count;
     GLfloat *verts = (GLfloat *) SDL_AllocateRenderVertices(renderer, vertlen, 0, &cmd->data.draw.first);
     int i;
+    SDL_Color color = {
+        .r = cmd->data.draw.r,
```

Comment:
Isn't this C99 syntax?
If SDL2 is still supposed to be C89-compatible this needs to be done differently

Reply:
Woops, I didn't realize that was C99+. I'll change that.